bin/print_targets.ml

```diff
@@ -0,0 +1,27 @@
+open! Stdune
+open Import
+open Fiber.O
+
+let doc = "Print available targets"
+
+let man =
+  [ `S "DESCRIPTION"
+  ; `P {|$(b,dune targets) print the available targets|}
+  ; `Blocks Common.help_secs
+  ]
+
+let info = Term.info "targets" ~doc ~man
+
+let term =
+  let%map common = Common.term in
+  Common.set_common common ~targets:[] ;
+  let log = Log.create common in
+  Scheduler.go ~log ~common (fun () ->
+      Import.Main.setup ~log common ~external_lib_deps_mode:true
+      >>= fun setup ->
+      let targets = Target.targets_of_path setup Path.root in
+      List.iter targets ~f:print_endline ;
+      Fiber.return ()
```

Comment:
minor, you should be able to remove this by using `>>|` above.

Reply:
I'm not sure that's possible.